packages/frontend-2/components/dashboard/Sidebar.vue

```diff
@@ -273,4 +261,19 @@ const openFeedbackDialog = () => {
   showFeedbackDialog.value = true
   isOpenMobile.value = false
 }
+
+const openWorkspaceCreateDialog = () => {
+  showWorkspaceCreateDialog.value = true
+  mixpanel.track('Create Workspace Button Clicked', {
```

Comment:
Hmmm. Yeah not ideal event. But let's just keep it for now and we'll make sure to filter for the source when doing analysis.

Reply:
Or I'm not sure actually. You're probably right about changing this now so that we can properly track the before and after of changing this.

Links to the explainer page could instead trigger `Clicked Link to Workspace Explainer` and this should also be triggered when clicking the nav item in the sidebar.